msgcntr/messageforums-app/src/webapp/jsp/discussionForum/message/moveThreadPicker.jsp

```diff
@@ -2,8 +2,9 @@
         <div class="selected-threads-to-move">
             <div>
                 <h:outputText value="#{msgs.move_thread_info1}"  />
-                <h:outputText styleClass="sourcetitle" escape="true" value="#{ForumTool.selectedTopic.topic.title}"  /> 
-                 <h:outputText value="#{msgs.move_thread_info2}"  />
+                <h:outputText styleClass="sourcetitle" escape="true" value=" #{ForumTool.selectedTopic.topic.title} "  />
+                <h:outputText value="#{msgs.move_thread_info2}"  />
+                <span id="sourcetopicid" data-id='<h:outputText value="#{ForumTool.selectedTopic.topic.id}" />' hidden></span>

```

Comment:
An id should be unique, id="sourcetopicid" doesn't seem very unique it actually might be better as a class and if you going to use a span or a div it should be used as a container, i.e.
```
<span id="'<h:outputText value="#{ForumTool.selectedTopic.topic.id}" />' class="sourcetopicid">
    <h:outputText styleClass="sourcetitle" escape="true" value=" #{ForumTool.selectedTopic.topic.title} "  />
    <h:outputText value="#{msgs.move_thread_info2}"  />
</span>

Reply:
Thanks @ern, as you suggested, I moved the span to be a container but added an id like id="sourcetopicid-XX", because that seemed more meaningful than e.g. id=1; id=2, etc. Also, the jquery selector for a class is similar to "attribute that begins with" in that it returns an array, so either way, I'd have to select/assume the first element of the array is the one we want (it _should_ always be an array with size = 1)